awscli/topics/config-vars.rst

```diff
@@ -81,6 +83,12 @@ The valid values of the ``output`` configuration variable are:
 * table
 * text
 
+``cli_timestamp_format`` controls the format of timestamps displayed by the AWS CLI.
+The valid values of the ``cli_timestamp_format`` configuration varaible are:
+
+* none - Display the timestamp exactly as recieved.
```

Comment:
A little ambiguous. May want to include wording about it being the unmodified value received over the wire or from HTTP response.